### PR TITLE
Cyhy feeds/remove cyhy core dependency

### PR DESCRIPTION
**Pull request by mcdonnnj**
This change is to support the new requirement for mongo-db-from-config needed by https://github.com/cisagov/cyhy-feeds/pull/26 to address https://github.com/cisagov/cyhy-feeds/issues/17

### CODE REVIEW

**Comment by jsf9k** [comment]
I'll take a look at the Travis builds and see why it's failing.

**Review by jsf9k — APPROVED** [review]
These changes look good to me.  You can go ahead and merge this after cisagov/cyhy-feeds#26 is merged.

**Review by dav3r — APPROVED** [review]
💪 💼 @mcdonnnj 👍

**Comment by KyleEvers** [comment]
LGTM

**Pull request merged**
